api/model.py

```diff
@@ -26,18 +23,16 @@ def get(self):
         """Return the metadata associated with the model"""
         return MODEL_META_DATA
 
-label_prediction = api.model('LabelPrediction', {
-    'embedding': fields.List(fields.List(fields.Float,required=True))
-})
 
 predict_response = api.model('ModelPredictResponse', {
     'status': fields.String(required=True, description='Response status message'),
-    'predictions': fields.List(fields.Nested(label_prediction), description='Predicted labels and probabilities')
+    'embedding': fields.List(fields.List(fields.Float, required=True, description="Generated embedding"))
```

Comment:
Will everyone know what `embedding` means here?

Reply:
I think so. The name of the model is Audio Embedding Generator